### DISCUSSION TITLE
Updating from v1.x

### DISCUSSION THREAD

**Discussion by jbonnet69** [mp-se/fermentrack-docker#45] [General]
Hello @mp-se,
Could you please tell me how to update to v2 from a v1 on a Synology NAS?
Thanks in advance
Julien

**Comment by mp-se** [comment]
There is short instruction in the docs, https://fermentrack-docker.readthedocs.io/

You need to update the volume mount points, the port and set the secret key as an environment setting. 

If you have issues i can make a few screendumps or write a more detailed instruction. 

Make a copy of the files in the db folder before upgrading just to be on the safe side.
